src/test/java/org/galatea/starter/entrypoint/ConstraintViolationMessageFormatterTest.java

```diff
@@ -0,0 +1,36 @@
+package org.galatea.starter.entrypoint;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+
+public class ConstraintViolationMessageFormatterTest {
+
+  @Test
+  public void violationMessagesConcatenated() {
```

Comment:
Do you have any error scenarios for this class? Should it be testing any non-ideal path? Check code coverage report.